RichEditorView/Classes/RichEditorView.swift

```diff
@@ -439,6 +445,35 @@ extension RichEditorView: UIGestureRecognizerDelegate {
 // MARK: - Utilities
 extension RichEditorView {
     
+    /** Sets content offset for scrollView based on caret location **/
+    
+    private func fixScrollView(editor: RichEditorView) {
+        
+        var scrollView = editor.webView.scrollView
+        
+        var contentHeight: CGFloat?
+        let htmlHeight = editor.runJS("document.getElementById('editor').clientHeight;")
```

Comment:
Generally a good practice to wrap the JS implementation details with a public accessor method on the RE namespace IMHO